src/confcom/azext_confcom/tests/latest/test_confcom_arm.py

```diff
@@ -5293,4 +5293,142 @@ def test_arm_template_security_context_seccomp_profile_missing_syscalls(self):
+            },
+            "memoryInGb": {
+            "type": "string",
+            "metadata": {
+                "description": "The amount of memory to allocate to the container in gigabytes."
+            },
+            "defaultValue": "1.5"
+            },
+            "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+            }
+        },
+        "resources": [
+            {
+            "name": "[parameters('containergroupname')]",
+            "type": "Microsoft.ContainerInstance/containerGroups",
+            "apiVersion": "2022-04-01-preview",
+            "location": "[parameters('location')]",
+            "properties": {
+                "containers": [
+                {
+                    "name": "[parameters('containername')]",
+
+                    "properties": {
+                    "image": "[variables('image')]",
+                    "command": [
+                        "python3"
+                    ],
+                    "ports": [
+                        {
+                        "port": "[parameters('port')]"
+                        }
+                    ],
+                    "resources": {
+                        "requests": {
+                        "cpu": "[parameters('cpuCores')]",
+                        "memoryInGb": "[parameters('memoryInGb')]"
+                        }
+                    }
+
+                }
+                }
+
+                ],
+
+                "osType": "Linux",
+                "restartPolicy": "OnFailure",
+                "confidentialComputeProperties": {
+                "IsolationType": "SevSnp"
+                },
+                "ipAddress": {
+                "type": "Public",
+                "ports": [
+                    {
+                    "protocol": "Tcp",
+                    "port": "[parameters( 'port' )]"
+                    }
+                ]
+                }
+            }
+            }
+        ],
+        "outputs": {
+            "containerIPv4Address": {
+            "type": "string",
+            "value": "[reference(resourceId('Microsoft.ContainerInstance/containerGroups/', parameters('containergroupname'))).ipAddress.ip]"
+            }
+        }
+    }
+    """
+    aci_policy = None
+
+    @classmethod
+    def setUpClass(cls):
+
+        cls.aci_arm_policy = load_policy_from_arm_template_str(cls.custom_arm_json, "")[
+            0
+        ]
+        cls.aci_arm_policy.populate_policy_content_for_all_images()
+
+    def test_stop_signal(self):
+        regular_image_json = json.loads(
+            self.aci_arm_policy.get_serialized_output(
+                output_type=OutputType.RAW, rego_boilerplate=False
+            )
+        )
+        # check for the signal for SIGQUIT. this is part of the nginx image
+        self.assertTrue(
+            3 in
```

Comment:
I would also check that it gets converted to a string properly by checking that SIGQUIT is in the policy

Reply:
I think it's only supposed to be a number in the policy

Reply:
I thought Bryce had said the OCI policy spec only considered the strings during standup

Reply:
[looks like you're right](https://github.com/opencontainers/image-spec/blob/b74f37875da0de3a68497790c0c74a996c8a6211/schema/config_test.go#L185C33-L185C33). I'll test to see if Docker does that translation for you or if it'll put a number in the image manifest

Reply:
talked with Matthew and he said the policy is expecting a number for the signal, not a name. I think we should be good as-is